test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -296,6 +297,9 @@ func BenchmarkPerfScheduling(b *testing.B) {
 						defer featuregatetesting.SetFeatureGateDuringTest(b, utilfeature.DefaultFeatureGate, feature, flag)()
 					}
 					dataItems.DataItems = append(dataItems.DataItems, runWorkload(b, tc, w)...)
+					// Reset metrics to prevent metrics generated in current workload gets
+					// carried over to the next workload.
+					legacyregistry.Reset()
```

Comment:
what I meant is `Reset` is not always needed, this will reset `testcase*workload` times while most of `reset` here as far as I can see is not necessary.

Reply:
Yes, it's discussed here: https://github.com/kubernetes/kubernetes/pull/99472#discussion_r584435795